filebeat/tests/system/test_registrar_upgrade.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+"""Test the registrar with old registry file formats"""
```

Comment:
?

Each method in the test file contains a 'version'. The version is the last beats version a registry file format has been used. That is, the tests check all different versions we introduced since 6.0. The bigger issue we have is the regsitry file format in general. No versioning is foreseen. This makes the process quite cumbersome. For this reason I will not allow any more changes to the current regsitry format + The new format will be versioned.